lua/boo/init.lua

```diff
@@ -32,20 +37,35 @@ function M.boo()
 		})
 	end
 	if config.close_on_mouse_move then
-		vim.api.nvim_create_autocmd({ 'CursorMoved', 'CursorMovedI' }, {
+		vim.api.nvim_create_autocmd({ 'CursorMoved', 'CursorMovedI', 'InsertEnter' }, {
 			buffer = 0,
 			desc = 'Closes boo when moving the cursor',
 			group = vim.api.nvim_create_augroup('Closeboo', { clear = true }),
 			callback = close_callback,
 		})
 	end
+	if config.open_on_mouse_hold then
+		vim.api.nvim_create_autocmd({ 'CursorHold', 'InsertLeave' }, {
+			buffer = 0,
+			desc = 'Open boo when holding cursor',
+			group = vim.api.nvim_create_augroup('Openboo', { clear = true }),
+			callback = open_callback,
+		})
+	end
+
+	-- vim.api.nvim_create_autocmd({ 'WinLeave', 'BufWinLeave', 'BufDelete', 'ModeChanged', 'InsertEnter' }, {
+	-- 	buffer = 0,
+	-- 	desc = 'Closes boo on various conditions',
+	-- 	group = vim.api.nvim_create_augroup('ClosebooOnEvents', { clear = true }),
+	-- 	callback = close_callback,
+	-- })
+
 	vim.api.nvim_set_option_value('buftype', 'nofile', { buf = boo_buffer })
 	vim.api.nvim_set_option_value('modifiable', false, { buf = boo_buffer })
 	vim.api.nvim_set_option_value('readonly', true, { buf = boo_buffer })
 
 	local lsp_info = get_lsp_info()
 	if #lsp_info <= 0 then
-		vim.notify('No info available', vim.log.levels.INFO, { title = 'boo' })
```

Comment:
This should definitely not be removed. If a user invokes boo but nothing happens, it will look like a bug. But if we display this info, it's clear what happened.

Reply:
we should remove it because not everytime we have information so the users know when to do this